webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
+  CleanWebpackPlugin = require('clean-webpack-plugin'),
+  HtmlWebpackPlugin = require('html-webpack-plugin'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const extractPlugin = new ExtractTextPlugin({filename: './assets/css/app.css'});
```

Comment:
```suggestion
const extractPlugin = new ExtractTextPlugin({ filename: './assets/css/app.css' });
```

space inside curly braces